tools/logger/logger.c

```diff
@@ -196,6 +198,9 @@ int main(int argc, char *argv[])
 			config.version_fw = 1;
 			config.version_file = optarg;
 			break;
+		case 'd':
```

Comment:
Can you make sure all these options are listed in the help.

Reply:
I didn't notice any lack in help message:
flags: `ho:i:l:ps:c:u:tev:r`
```
Usage sof-logger <option(s)> <file(s)>
sof-logger:                             Display mailbox contents
sof-logger:      -i infile -o outfile   Dump infile contents to outfile
sof-logger:      -l *.ldc_file          *.ldc files generated by rimage
sof-logger:      -p                     Input from stdin
sof-logger:      -e                     Enable checking firmware version with default verification file
sof-logger:      -v ver_file            Enable checking firmware version with ver_file file
sof-logger:      -c                     Set timestamp clock in MHz
sof-logger:      -s state_name          Take a snapshot of state
sof-logger:      -t                     Display trace data
sof-logger:      -u baud                Input data from a UART
sof-logger:      -r                     Less formatted output for chained log processors
sof-logger:      -d                     Dump ldc information
```

Reply:
The help for "c:" isn't exactly proper as flags expects an argument but that isn't mentioned in the help explicitly.

Reply:
added